kube-client/src/config/file_loader.rs

```diff
@@ -66,33 +66,34 @@ impl ConfigLoader {
         } else {
             return Err(KubeconfigError::CurrentContextNotSet);
         };
+
```

Comment:
Let's revert this file as well because these are not affected by the changes.

Reply:
it actually is affected, because context, cluster and user are now `Option<T>`